tests/tests_configure_ha_cluster.yml

```
@@ -3,7 +3,8 @@
 - name: Verify the role on the primary node
   hosts: all
   vars:
-    __mssql_single_node_test: true
+    __mssql_single_node_test: "{{
```

```suggestion
    __mssql_single_node_test: "{{ ansible_play_hosts_all | length <= 1 }}"
```
or `length == 1` if you don't care if the length is 0

---

how `ansible_play_hosts_all` can be 0?

---

> how `ansible_play_hosts_all` can be 0?

I don't think it can be so `length == 1` is ok